libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/ExecSpecRule/Conway/Base.hs

```diff
@@ -162,10 +162,11 @@ instance
       runGenRS proof genSize $
         genAlonzoTx proof ueSlot
     ueCertState <- arbitrary
+    let txSize = uecTx ^. sizeTxF
     let
       uePParams =
         gePParams (gsGenEnv gs)
-          & ppMaxTxSizeL .~ 3000
+          & ppMaxTxSizeL .~ fromIntegral txSize
```

Comment:
This is not safe, but assuming that the generated transaction is smaller than 4GB.. sounds like a reasonable assumption, no?

Reply:
Yes, it is safe to assume generated transactions are smaller than 4Gb.